ansible_collections/ds/ansible_ds/tests/conftest.py

```diff
@@ -71,7 +71,7 @@ def isPrefix(dir):
         except PermissionError:
             return False
 
-    def setPrefix(dir, lib389dir):
+    def setPrefix(dir):
```

Comment:
IMHO we should have a test here with 2 paths:
if dir == '/':
   os.environ.pop('PREFIX', None)
   lp= '/usr' + lp
else:
   os.environ['PREFIX'] = dir